src/app/garfile/view/post.controller.js

```diff
@@ -35,18 +38,17 @@ module.exports = (req, res) => {
         garpeople: parsedPeople,
         supportingdocs: supportingDocuments,
       };
+      renderContext.showChangeLinks = true;
       if ((parsedGar.status.name === 'Submitted') || parsedGar.status.name === 'Cancelled') {
-        logger.info('Rendering submitted GAR review page');
```

Comment:
Removed the submitted and un-submitted pages as there is not really any difference, nothing that the logic already could handle in a single page.